.travis.yml

```diff
@@ -0,0 +1,180 @@
+  - PORTAGE=2.2.15 CATEGORY=media-video
+  - PORTAGE=2.2.15 CATEGORY=net-analyzer
+  - PORTAGE=2.2.15 CATEGORY=net-dialup
+  - PORTAGE=2.2.15 CATEGORY=net-dns
+  - PORTAGE=2.2.15 CATEGORY=net-firewall
+  - PORTAGE=2.2.15 CATEGORY=net-fs
+  - PORTAGE=2.2.15 CATEGORY=net-ftp
+  - PORTAGE=2.2.15 CATEGORY=net-im
+  - PORTAGE=2.2.15 CATEGORY=net-irc
+  - PORTAGE=2.2.15 CATEGORY=net-libs
+  - PORTAGE=2.2.15 CATEGORY=net-mail
+  - PORTAGE=2.2.15 CATEGORY=net-misc
+  - PORTAGE=2.2.15 CATEGORY=net-nds
+  - PORTAGE=2.2.15 CATEGORY=net-news
+  - PORTAGE=2.2.15 CATEGORY=net-nntp
+  - PORTAGE=2.2.15 CATEGORY=net-p2p
+  - PORTAGE=2.2.15 CATEGORY=net-print
+  - PORTAGE=2.2.15 CATEGORY=net-proxy
+  - PORTAGE=2.2.15 CATEGORY=net-voip
+  - PORTAGE=2.2.15 CATEGORY=net-wireless
+  - PORTAGE=2.2.15 CATEGORY=perl-core
+  - PORTAGE=2.2.15 CATEGORY=rox-base
+  - PORTAGE=2.2.15 CATEGORY=rox-extra
+  - PORTAGE=2.2.15 CATEGORY=sci-astronomy
+  - PORTAGE=2.2.15 CATEGORY=sci-biology
+  - PORTAGE=2.2.15 CATEGORY=sci-calculators
+  - PORTAGE=2.2.15 CATEGORY=sci-chemistry
+  - PORTAGE=2.2.15 CATEGORY=sci-electronics
+  - PORTAGE=2.2.15 CATEGORY=sci-geosciences
+  - PORTAGE=2.2.15 CATEGORY=sci-libs
+  - PORTAGE=2.2.15 CATEGORY=sci-mathematics
+  - PORTAGE=2.2.15 CATEGORY=sci-misc
+  - PORTAGE=2.2.15 CATEGORY=sci-physics
+  - PORTAGE=2.2.15 CATEGORY=sci-visualization
+  - PORTAGE=2.2.15 CATEGORY=sec-policy
+  - PORTAGE=2.2.15 CATEGORY=sys-apps
+  - PORTAGE=2.2.15 CATEGORY=sys-auth
+  - PORTAGE=2.2.15 CATEGORY=sys-block
+  - PORTAGE=2.2.15 CATEGORY=sys-boot
+  - PORTAGE=2.2.15 CATEGORY=sys-cluster
+  - PORTAGE=2.2.15 CATEGORY=sys-devel
+  - PORTAGE=2.2.15 CATEGORY=sys-firmware
+  - PORTAGE=2.2.15 CATEGORY=sys-freebsd
+  - PORTAGE=2.2.15 CATEGORY=sys-fs
+  - PORTAGE=2.2.15 CATEGORY=sys-infiniband
+  - PORTAGE=2.2.15 CATEGORY=sys-kernel
+  - PORTAGE=2.2.15 CATEGORY=sys-libs
+  - PORTAGE=2.2.15 CATEGORY=sys-power
+  - PORTAGE=2.2.15 CATEGORY=sys-process
+  - PORTAGE=2.2.15 CATEGORY=virtual
+  - PORTAGE=2.2.15 CATEGORY=www-apache
+  - PORTAGE=2.2.15 CATEGORY=www-apps
+  - PORTAGE=2.2.15 CATEGORY=www-client
+  - PORTAGE=2.2.15 CATEGORY=www-misc
+  - PORTAGE=2.2.15 CATEGORY=www-plugins
+  - PORTAGE=2.2.15 CATEGORY=www-servers
+  - PORTAGE=2.2.15 CATEGORY=x11-apps
+  - PORTAGE=2.2.15 CATEGORY=x11-base
+  - PORTAGE=2.2.15 CATEGORY=x11-drivers
+  - PORTAGE=2.2.15 CATEGORY=x11-libs
+  - PORTAGE=2.2.15 CATEGORY=x11-misc
+  - PORTAGE=2.2.15 CATEGORY=x11-plugins
+  - PORTAGE=2.2.15 CATEGORY=x11-proto
+  - PORTAGE=2.2.15 CATEGORY=x11-terms
+  - PORTAGE=2.2.15 CATEGORY=x11-themes
+  - PORTAGE=2.2.15 CATEGORY=x11-wm
+  - PORTAGE=2.2.15 CATEGORY=xfce-base
+  - PORTAGE=2.2.15 CATEGORY=xfce-extra
+
+before_script:
+    - sudo chmod 777 /etc/passwd /etc/group /etc /usr /usr/bin
+    - echo "portage:x:250:250:portage:/var/tmp/portage:/bin/false" >> /etc/passwd
+    - echo "portage::250:portage,travis" >> /etc/group
+    - mkdir -p /etc/portage/ /usr/portage/distfiles
+    - mv * /usr/portage
```

Comment:
Yes, I checked this again, Basically `repos.conf` refers to `/usr/portage` and `repoman` wants to download `metadata.dtd` into `/usr/portage/distfiles`.